src/clients/c++/image_client.cc

```diff
@@ -490,6 +549,16 @@ FileToInputData(
 
   // Pre-process the image to match input size expected by the model.
   Preprocess(img, format, type1, type3, c, cv::Size(w, h), scale, input_data);
+
+  if (use_shm) {
+    *byte_size = (size_t)(img.total() * img.elemSize());
+
+    // place image data into memory in the form [channel][width][height]
```

Comment:
This comment isn't generally true since image_client supports both CHW and HWC.

Reply:
Removed